### DISCUSSION TITLE
Getting a MySQL error "Data too long for column 'event' at row 1"

### DISCUSSION THREAD

**Discussion by prha** [dagster-io/dagster#18313] [Q&A] [answered]
When materializing an asset with metadata of a certain size, I'm getting the following error:
```
sqlalchemy.exc.DataError: (mysql.connector.errors.DataError) 1406 (22001): Data too long for column 'event' at row 1
```

How can I fix this?

**Comment by prha** [comment, marked as answer]
This occurs because the `event_logs` table in MySQL stores the event metadata in a column of type `TEXT` instead of `LONGTEXT`.

Starting in `1.5.10`, new Dagster instances on MySQL should use `LONGTEXT` instead of `TEXT`, but existing instances will have to manually migrate.  This is to allow existing instances who aren't running into this to avoid a schema migration that could potentially incur a long downtime.

To migrate manually, you should run the following `ALTER` statements:
`ALTER TABLE event_logs MODIFY event LONGTEXT;`;
`ALTER TABLE asset_keys MODIFY last_materialization LONGTEXT;`

**Reply by zee-mel-2022** [reply]
Thanks @prha

**Reply by Marmikiih** [reply]
thanks @prha it works
